userguide/install.rst

```diff
@@ -33,19 +33,88 @@ Getting %brand%
 -------------------------
 
 The latest STABLE version of %brand% |release| is available for download
-from `<https://download.freenas.org/11.2/STABLE/latest/>`__.
+from `<https://www.freenas.org/download-freenas-release/>`__.
 
-.. note:: %brand% requires 64-bit hardware.
+The download page has links to %brand% release notes, :file:`.iso`
+integrity checksums, and PGP security keys.
+
+Clicking :guilabel:`Download` opens a dialog to save an *.iso* file.
+This bootable installer should be
+:ref:`written to physical media <Preparing the Media>` before installing
+%brand%.
+
+
+.. index:: Verify download files
+
+Checking Installer Integrity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+%brand% uses the
+`OpenPGP standard <https://en.wikipedia.org/wiki/Pretty_Good_Privacy#OpenPGP>`__
+to show that downloadable files have been provided by a trustworthy
+source. Verifying the PGP signature of a %brand% file requires OpenPGP
+compliant encryption software like
+`gnupg <https://www.freebsd.org/cgi/man.cgi?query=gpg>`__,
+`Kleopatra <https://www.openpgp.org/software/kleopatra/>`__,
+or `Gpg4win <https://gpg4win.org/>`__.
+
+The :file:`sha256.txt` file is used to confirm the integrity of the
```

Comment:
If this is an introduction to the usage of the sha256 file, it should mention that details are shown below.  Otherwise, it can be integrated with that section.